Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -163,6 +163,15 @@ bool EventHandler::handle_mousewheel(CSSPixelPoint position, unsigned button, un
     if (auto result = target_for_mouse_position(position); result.has_value())
         paintable = result->paintable;
 
+    auto* containing_block = paintable->containing_block();
+    while (containing_block) {
+        if (containing_block->is_scrollable()) {
+            const_cast<Painting::PaintableBox*>(containing_block->paintable_box())->handle_mousewheel({}, position, buttons, modifiers, wheel_delta_x * 24, wheel_delta_y * 24);
```

Comment:
This should use the `scroll_step_size` constant that's defined in this function: 
```suggestion
            const_cast<Painting::PaintableBox*>(containing_block->paintable_box())->handle_mousewheel({}, position, buttons, modifiers, wheel_delta_x * scroll_step_size, wheel_delta_y * scroll_step_size);
```

Reply:
Though it does look a little odd that here it's multiplied by the step size, but not for the other `handle_mousewheel()` call in this function.